packages/@adobe/spectrum-css-temp/components/menu/skin.css

```diff
@@ -29,6 +29,8 @@ governing permissions and limitations under the License.
 .spectrum-Menu {
   --spectrum-heading-subtitle3-text-color: var(--spectrum-global-color-gray-900);
   background-color: var(--spectrum-selectlist-background-color);
+  /* Define this var so it can be read from JS */
+  --spectrum-submenu-offset-distance-x: var(--spectrum-global-dimension-size-50);
```

Comment:
Should be border radius + 1 (size-65) which we don't have. I'll update the naming, but I didn't want to do the `calc` in the CSS because getComputedSize + getPropertyValue will return the string rather than the calculated version

Reply:
that's interesting. can you put in a comment with the calc so we know what it's derived from?